recipes/street_easy/build.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import pandas as pd
+
+# Read input data
+cols = [
+        "nta_name",
+        "nta_code",
+        "no._of_new_sale_listings",
+        "no._of_pending_sale_listings",
+        "no._of_sale_listings",
+        "pct_hike_(sales)",
+        "pct_cut_(sales)",
+        "median_weeks_on_market_(sales)",
+        "no._of_new_rental_listings",
+        "no._of_pending_listings",
+        "no._of_rental_listings",
+        "pct_hike_(rentals)",
+        "pct_cut_(rentals)",
+        "pct_furnished_(rentals)",
+        "pct_short_term_(rentals)",
+        "pct_with_concessions_(rentals)",
+        "median_weeks_on_market_(rentals)",
+        "week_start_date",
+        "week_end_date"
+    ]
+
+df = pd.read_csv("https://streeteasy-market-data-download.s3.amazonaws.com/nta/nta-metrics-2020-06-01.csv")
```

Comment:
should we expose this url? or at least hide `https://streeteasy-market-data-download.s3.amazonaws.com`